03-spatial-joins.ipynb

```diff
@@ -239,6 +239,92 @@
     "joined['continent'].value_counts()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "**Exercise**: Which rivers pass within 1 degree of Paris? And for Hong Kong?\n",
```

Comment:
I would maybe do it in meters (that's often problem that you need to convert the crs for that)

Reply:
I tried first but I ran into issues trying to get a world CRS in metres that worked and the `rivers` layer was acting up so, for the sake of the example, I left it on degrees.